File: core/src/test/java/w/commander/execution/TestCommandExecutionContext.java
package w.commander.execution;

import w.commander.CommandSender;
import w.commander.RawCommandArguments;

/**
 * @author whilein
 */
public class TestCommandExecutionContext extends AbstractCommandExecutionContext {

    public TestCommandExecutionContext(
            CommandSender sender,
            CommandExecutor executor,
            RawCommandArguments rawArguments
    ) {
        super(sender, executor, rawArguments);
    }

}

File: tests/src/main/kotlin/w/commander/tests/TestCommandExecutionContext.kt
package w.commander.tests

import w.commander.CommandSender
import w.commander.RawCommandArguments
import w.commander.execution.AbstractCommandExecutionContext
import w.commander.execution.CommandExecutor

/**
 * @author whilein
 */

class TestCommandExecutionContext(
        sender: CommandSender,
        executor: CommandExecutor,
        rawArguments: RawCommandArguments
) : AbstractCommandExecutionContext(sender, executor, rawArguments)

